src/main/java/eu/darkbot/api/managers/KeyBindsAPI.java

```diff
@@ -24,6 +26,18 @@
     @Nullable
     Character getKeyBind(@NotNull Shortcut shortcut, int slotNumber) throws IllegalArgumentException;
 
+    /**
+     * Search {@link Character} associated with given slotEntry
+     *
+     * @param slotEntry with {@link Shortcut} as key and {@link Integer} as value
+     * @return {@link Character} associated with given {@link Map.Entry} otherwise null
+     * @throws IllegalArgumentException if {@code slotEntry.value < 1 || slotEntry.value > 10}
+     */
+    @Nullable
+    default Character getKeyBind(@NotNull Map.Entry<SlotBarAPI.Type, Integer> slotEntry) throws IllegalArgumentException {
```

Comment:
this is a very weird way of passing parameters, i wouldn't make it part of the API